test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/VsProjectProperty.cs

```diff
@@ -15,7 +15,7 @@ public VsProjectProperty(string name, string value)
         {
             if (string.IsNullOrEmpty(name))
             {
-                throw new ArgumentException(ProjectManagement.Strings.Argument_Cannot_Be_Null_Or_Empty, nameof(name));
+                throw new ArgumentException("Argument cannot be null or empty", nameof(name));
```

Comment:
I'll leave it upto you to take a call. But i think It's more convenient to use resource strings even in tests then when string change we don't need do update all the tests again. Currently, I see lot of hard coded strings in our most tests to verify error/ warning messages.